### PR TITLE
Add PapapiuLab N1 3D printer

### PR DESCRIPTION
**Pull request by papapiulab**
Dear Prusa Team,

I am writing on behalf of PapapiuLab, a team brand new design team to providing Stable High Temperature Materials 3D printing solutions. We have recently developed frist printer profiles for the PrusaSlicer software and would like to request its inclusion in the official PrusaSlicer profile Github repository.

We have thoroughly tested and validated our profile and believe that it would be a valuable addition to the existing collection. Our profile has been designed to optimize the performance of our printers PapapiuLab N1 and to provide users with an enhanced printing experience. We believe that this profile will be beneficial for all PrusaSlicer users, especially those who use our printers.

PrusaSlicer is a very nice slicing software, and we sincerely thank your team for contributing to the 3D printing community.

Please let us know if you require any further information or testing of our profile.

Best regards,
PapapiuLab Design Team

### CODE REVIEW

File: live/PapapiuLab/0.1.0.ini [hunk 1]
```diff
@@ -0,0 +1,510 @@
+# Print profiles for Papapiu printers.
+# Based on the profiles from PR: https://github.com/slic3r/slic3r-profiles/pull/13 by top-gun.
```
Comment (rtyr) [inline live/PapapiuLab/0.1.0.ini:2]: This is not valid for this bundle. Remove or update this line.

File: live/PapapiuLab/0.1.0.ini [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+# The graphics for the bed (SVG) was created by PwnCNC and published on Thingiverse under the Creative Commons Attribution license: https://www.thingiverse.com/thing:4861532/files
```
Comment (rtyr) [inline live/PapapiuLab/0.1.0.ini:3]: This is not valid for this bundle. Remove or update this line.

File: live/PapapiuLab/0.1.0.ini [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+# The printer models will be shown by the Configuration Wizard in this order,
+# also the first model installed & the first nozzle installed will be activated after install.
+# Printer model name will be shown by the installation wizard.
+
+[printer_model:Papapiu_N1S]
+name = Papapiu N1S
+variants = 0.4
+technology = FFF
+bed_model = N1_Bed.stl
+bed_texture = N1_Texture.png
```
Comment (rtyr) [inline live/PapapiuLab/0.1.0.ini:20]: You have to either also add `thumbnail = N1_thumbnail.png` here or rename the thumbnail file to `Papapiu_N1S_thumbnail.png`.

**Review by rtyr — CHANGES_REQUESTED** [review]

File: live/PapapiuLab/0.1.0.ini (hunk 1 above)
Comment (papapiulab) [reply live/PapapiuLab/0.1.0.ini:2]: Thank u very much for your code review, we will fix it now.

**Comment by rtyr** [comment]
Your profiles are now available in PS2.6.0 and newer.

I resized the thumbnail (yours was 1800x2825).

**Comment by papapiulab** [comment]
> Your profiles are now available in PS2.6.0 and newer.
> 
> I resized the thumbnail (yours was 1800x2825).

Thank you for your help and support, salute!

**Pull request merged**